### DISCUSSION TITLE
[Feature] Use Frigate with Immich to detect faces

### DISCUSSION THREAD

**Discussion by mikealanni** [immich-app/immich#14787] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hi, Would Integrate Immich with Frigate to detect faces and do some automation on home assistant of faces. Such if my camera sees me and one of my family member open the door etc. 

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Previously requested in #12916
